classes/gui/ScatterView.sc

```diff
@@ -50,19 +90,18 @@ ScatterView {
 	resize_{arg resize;
 		plot.resize_(resize)
 	}
-	initPlot {
```

Comment:
should we standardise the `arg` list here? i.e. make it consistent across methods to be either `|a, b, c|` or `arg a, b, c;`?

Reply:
sure, I see |args...| used more widely throughout the code. I only really use the arg syntax when I'm teaching because pipes are a pain for people with German keyboards.